feat(as): AS configuration resource support new fields (#4923)
huaweicloud/terraform-provider-huaweicloud: docs/resources/as_configuration.md

* `ecs_group_id` - (Optional, String, ForceNew) Specifies the ECS group ID. Changing this will create a new resource.

* `tenancy` - (Optional, String, ForceNew) Configure this field to **dedicated** to create ECS instances on DeHs.
  Before configuring this field, prepare DeHs. Changing this will create a new resource.

* `dedicated_host_id` - (Optional, String, ForceNew) Specifies the ID of the DEH.
  Changing this will create a new resource.

> -> This parameter is valid only when `tenancy` is set to **dedicated**.

Reviewer: ```suggestion
  -> This parameter is valid only when `tenancy` is set to **dedicated**.
```

* `ecs_group_id` - (Optional, String, ForceNew) Specifies the ECS group ID. Changing this will create a new resource.

> * `tenancy` - (Optional, String, ForceNew) Specifies creating an ECS instance on the DEH (dedicated host).

Reviewer: ```suggestion
* `tenancy` - (Optional, String, ForceNew) Configure this field to **dedicated** to create ECS instances on DeHs. Before configuring this field, prepare DeHs. Changing this will create a new resource.
```

And remove the follow descriptions.

Author: done

  Changing this will create a new resource.

  -> The parameter is mandatory when creating an ECS instance on the DEH, its value is **dedicated**.
  Otherwise, this parameter can be ignored.

* `dedicated_host_id` - (Optional, String, ForceNew) Specifies the ID of the DEH.
  Changing this will create a new resource.

  -> The parameter is valid only when `tenancy` is set to **dedicated**.
  If the field is specified, the ECS instance will be created on the specified DEH.
  If the field is not specified, the ECS instance will be created on the DEH that has the largest remaining memory
  among all DEHs that meet the specifications requirement to achieve load balancing. In this case, automatic placement

>   must be enabled for at least one DEH.

Reviewer: ```suggestion
  -> This parameter is valid only when `tenancy` is set to **dedicated**.
  <br/>If this parameter is specified, ECSs will be created on a specified DeH.
  <br/>If this parameter is not specified, the system automatically selects the DeH with the maximum
  available memory size from the DeHs that meet specifications requirements to create the ECSs, thereby
  balancing load of the DeHs.
```

Author: done

* `volume_type` - (Required, String, ForceNew) Specifies the disk type. Changing this will create a new resource.
  Available options are:
  + **SAS**: High I/O EVS disk.
  + **GPSSD**: General-purpose SSD EVS disk.
  + **SSD**: Ultra-high I/O EVS disk.
  + **ESSD**: High-speed I/O EVS disk.
  + **GPSSD2**: General-purpose SSD V2 EVS disk.
  + **ESSD2**: High-speed SSD V2 EVS disk.

> 

Reviewer: Please refer to the English documentation and modify the corresponding type description.

* `kms_id` - (Optional, String, ForceNew) Specifies the encryption KMS ID of the **DATA** disk.
  Changing this will create a new resource.

* `iops` - (Optional, Int, ForceNew) Specifies configuring IOPS for EVS disk.
  Changing this will create a new resource.

  -> If `volume_type` value is **GPSSD2** or **ESSD2**, the parameter is mandatory. If `volume_type` is other values,
  the parameter does not to be set.

* `throughput` - (Optional, Int, ForceNew) Specifies the throughput of the EVS disk, unit is MiB/s.
  Changing this will create a new resource.

  -> If `volume_type` value is **GPSSD2**, the parameter is mandatory. If `volume_type` is other values, the parameter
  does not to be set.

> 

Reviewer: Please refer to the English documentation and modify the fields `iops` and `throughput` description.

The `bandwidth` block supports:

* `share_type` - (Required, String, ForceNew) Specifies the bandwidth sharing type.

>   The value can be **PER** (exclusive bandwidth) and **WHOLE** (shared bandwidth).

Reviewer: ```suggestion
  The value can be **PER** (exclusive bandwidth) or **WHOLE** (shared bandwidth).
```

Author: done
